src/kv/rocksdb_cache/BinnedLRUCache.cc

```diff
@@ -460,6 +483,21 @@ size_t BinnedLRUCacheShard::GetPinnedUsage() const {
   return usage_ - lru_usage_;
 }
 
+void BinnedLRUCacheShard::rotate_bins() {
```

Comment:
Why `rotate`? If it were `age` we would have here the reason for "Age Binning" :D

Reply:
hrm, perhaps.  Age ties the idea more closely to the concept of age binning, but I like in some ways keeping the concept of bin rotation somewhat distinct from the concept of aging.  IE perhaps at some point in the future rotating the bins might not be triggered strictly by an aging event.  Do you feel strongly about changing the wording here?

Reply:
Maybe `shift_bins`?
My opposition to `rotate` is because there is nothing circular in bins.
But I will not insist on it.

Reply:
I can get behind shift_bins.  I'll make the change.